src/main/java/uk/gov/companieshouse/overseasentitiesapi/configuration/MongoConfig.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.companieshouse.overseasentitiesapi.configuration;
```

Comment:
Suggest adding a basic unit-test for this class. At the very least it will improve slightly the code coverage.

Reply:
Unit test added.